lib/pure/hashes.nim

```diff
@@ -282,8 +280,12 @@ proc hash*[T: tuple](x: T): Hash =
 
 proc hash*[A](x: openArray[A]): Hash =
   ## Efficient hashing of arrays and sequences.
-  for it in items(x): result = result !& hash(it)
-  result = !$result
+  when A is char|byte:
```

Comment:
- Do we add uint8 as well?

- Do we add other integers? And floats?

Reply:
uint8 is an alias for byte.